docs/detections/add-exceptions.asciidoc

```diff
@@ -59,6 +59,11 @@ image::images/add-exception-ui.png[]
 Add conditions that define when the exception prevents alerts:
 
   .. *Field*: Select a field to identify the event being filtered.
++  
+[NOTE] 
+=======
+Fields with conflicts are marked with a warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]). Using these fields and might cause unexpected exceptions behavior. For more information, refer to <<rule-exceptions-field-conflicts,Warning about type conflicts and unmapped fields>>.
```

Comment:
```suggestion
Fields with conflicts are marked with a warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]). Using these fields may cause unexpected exceptions behavior. For more information, refer to <<rule-exceptions-field-conflicts, troubleshooting type conflicts and unmapped fields>>.
```
The might/may thing is super trivial, feel free to ignore, but this inspired me to look up the difference and apparently "may" indicates a more strong possibility, which seems accurate in this case

Reply:
Thanks both for sharing your perspectives! @benironside I've actually been taught what @jmikell821 described and tend to use `might` when writing about a situation that can potentially happen. For now, I'd like to keep the text as is because I still think `might` works slightly better than `may` in this sentence, but am always open to following a different standard down the line if decide differently. 

```suggestion
Fields with conflicts are marked with a warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]). Using these fields and might cause unexpected exceptions behavior. For more information, refer to <<rule-exceptions-field-conflicts,Troubleshooting type conflicts and unmapped fields>>.
```